services/time/src/api.rs

```diff
@@ -12,7 +12,60 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-//! Exonum-time API.
+//! HTTP API for the time service. All APIs are accessible from the public HTTP server
+//! of the node.
```

Comment:
What about the private endpoint getting times for current validators? Without its description, the purpose of `ValidatorTime` type is not quite clear. I don't think private APIs are necessarily private *interfaces*; the split is about access / privilege level, not stability.